### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ChirashiSushi**
### Please review these reminders and either check the box or explain why not:

- [ ] Read and followed the [contributing guidelines](https://eds.eks.dev.ethos-int.com/#/Guidelines?id=section-contribute)?
- [ ] Component is exported from [index.js](https://github.com/getethos/ethos-design-system/blob/master/src/components/index.js)?
- [ ] Type is exported from [index.d.ts](https://github.com/getethos/ethos-design-system/blob/master/src/components/index.d.ts) so Typescript users can consume it? Added a test and ran `yarn test:types`?
- [ ] Bumped the yarn version?

### Referencing PR or Branch (e.g. in CMS or monorepo):

-

### CODE REVIEW

**Review by WesleyDRobinson — APPROVED** [review]
oops! I did not follow this update in my most recent change.
I tagged my PR branch and pushed that.

I'll follow this procedure next time.

File: README.md [hunk 1]
```diff
@@ -102,11 +102,13 @@ Each time you need to consume recent EDS changes elsewhere, you need to bump the
 - Prompt you for the new version number
 - Update `package.json` with the version number
 - Commit the change in a new commit
+- Get your PR approved in your branch and merge it to master
 
-2. To create and push a git tag, do one of these:
+2. To create and push a git tag:
```
Comment (coleEthos) [inline README.md:107]: This should just say `To push a git tag:`  


[IIUC docs](https://classic.yarnpkg.com/en/docs/cli/version#toc-git-tags) the tag is being created when you run `yarn version`.
> If you run yarn version within a Git repository an [annotated Git tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging) will be created by default following the format v0.0.0.

File: README.md [hunk 2]
```diff
@@ -102,11 +102,13 @@ Each time you need to consume recent EDS changes elsewhere, you need to bump the
...
 
-- Run `git tag v1.2.3 && git push origin v1.2.3` on your branch.
-- Create a release `v1.2.3` on GitHub, if it's already in trunk.
+- Switch to `master` branch and pull in your latest change
+- Run `git push origin v{new version number}` this will create the new tag with the version number in Github
+- Draft a new release in Github using the new tag
```
Comment (coleEthos) [inline README.md:111]: What's the value in doing these releases? They aren't technically required, only the package.json and git tag combined are needed by consumers of EDS.

File: README.md [hunk 3]
```diff
@@ -102,11 +102,13 @@ Each time you need to consume recent EDS changes elsewhere, you need to bump the
...
 - Update `package.json` with the version number
 - Commit the change in a new commit
+- Get your PR approved in your branch and merge it to master
 
-2. To create and push a git tag, do one of these:
+2. To create and push a git tag:
 
-- Run `git tag v1.2.3 && git push origin v1.2.3` on your branch.
-- Create a release `v1.2.3` on GitHub, if it's already in trunk.
+- Switch to `master` branch and pull in your latest change
```
Comment (coleEthos) [inline README.md:109]: I don't think this step is needed. As long as the last commit in your PR before it was merged, is the one where you ran `yarn version`, the tag should be on that commit. Pushing the tag doesn't require you be on any specific branch, does it?

File: README.md (hunk 2 above)
Comment (coleEthos) [reply README.md:111]: I know we already had it in the readme, not sure why we started doing it is all.

Comment (ChirashiSushi) [reply README.md:111]: I suppose creating a new release with the most recent changes since the previous release is more for bookkeeping?

Comment (coleEthos) [reply README.md:111]: Yeah fair enough. I guess the tags just seem to do that job to some extent as well already.

**Review by coleEthos — APPROVED** [review]
📖 ✅

**Pull request merged**